features/feature_case/asm/asm_cvt.cu

```diff
@@ -0,0 +1,60 @@
+// ====------ asm_cvt.cu ----------------------------------- *- CUDA -* ---===//
```

Comment:
We have https://github.com/oneapi-src/SYCLomatic-test/blob/1a9db7e9c27075a05ff763a32d1739d8a3a185fd/features/feature_case/asm/asm_arith.cu#L602  to test migration of PTX cvt, pls add the new test to there.